core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionRepository.java

```diff
@@ -37,11 +32,13 @@
  */
 @RequiredArgsConstructor
 public class BuiltinFunctionRepository {
-
   public static final String DEFAULT_NAMESPACE = "default";
 
   private final Map<String, Map<FunctionName, FunctionResolver>> namespaceFunctionResolverMap;
 
+  @Getter
```

Comment:
why need @Getter? it should not been exposed, right?

Reply:
Was used in a not very useful test.

Removed both the getter and the test in e7ccfc2bd2cc20c2c581210a58b5afda3ccf4d50.